readme.md

```diff
@@ -1,7 +1,7 @@
 # Hello world
 Replace `<your_account>` with your Github username and copy the links to Pull Request description:
-- [DEMO LINK](https://<your_account>.github.io/layout_hello-world/)
-- [TEST REPORT LINK](https://<your_account>.github.io/layout_hello-world/report/html_report/)
+- [DEMO LINK](https://Klymenko-Dmytro.github.io/layout_hello-world/)
+- [TEST REPORT LINK](https://Klymenko-Dmytro.github.io/layout_hello-world/report/html_report/)
```

Comment:
In the next pull paste these links here ⏬ 
<img width="491" alt="image" src="https://user-images.githubusercontent.com/94261494/172626098-b742c611-f0ee-4aef-b13e-ad67e4023258.png">